tests/lib/external/snapd-testing-tools/utils/log-parser

```diff
@@ -310,47 +254,38 @@ class LogReader:
                 prepared_log.append(item)
             else:
                 prepared_log.append(item.__dict__())
-        with open(filepath, 'w') as json_file:
+        with open(filepath, "w") as json_file:
             json.dump(prepared_log, json_file, indent=4)
 
     def _next_line(self):
```

Comment:
@zyga wrote: Why is this not just a iter(self.lines) somewhere? Please remove this.